rocky/reports/templates/summary/report_asset_overview.html

```diff
@@ -3,7 +3,7 @@
 <section id="asset-overview">
     <div>
         <h2>{% translate "Asset overview" %}</h2>
-        <h3>{% translate "Selected objects" %}({{ total_oois }})</h3>
+        <h3>{% translate "Selected objects" %} ({{ total_oois }})</h3>
```

Comment:
I have taken this up with UX and their conclusion was to leave it like this, for now. Reason being: since it's consistent with the title in the previous step, when selecting objects. And to add to that this flow will be revised eventually, including all texts.